src/config/settings.development.json

```diff
@@ -1,9 +1,9 @@
 {
   "environment": "development",
   "defaultLocale": "en-US",
-  "apiLocation": "https://kudo-o-matic-staging.dokku.kabisa.io/api/v1",
+  "apiLocation": "https://kudo-o-matic-staging.dokku.kabisa.io/api/v2",
   "authorizationLocation": "https://kudo-o-matic-staging.dokku.kabisa.io",
-  "client_id": "<client_id>",
-  "client_secret": "<client_secret>",
+  "client_id": "d1f64137a2ae8c084d99d61dc9c9fab1bf960dea136580098f8654e9e1cc0b00",
```

Comment:
It's a bad practice to store any kind of credential under version control - even for development. How we normally handle this is through one of two solutions:

 1. Use Environment variables. But since this is a JSON file, it's not possible to use this approach. 
 2. Use a sample file (e.g. `settings.example.json`) with the placeholders; all real configuration files are then ignored from version control (in `.gitignore`). This way, each developer copies the example json and fills out the configuration. The same goes for production. It may be necessary that a CI job generates the configuration file on the fly. 

Bottom line: never, ever store credentials in git!